core/domain/exp_domain.py

```diff
@@ -1289,7 +1296,8 @@ def get_all_html_content_strings(self):
             solution_html = self.solution.explanation.html
             html_list = html_list + [solution_html]
 
-        if self.id in ('ItemSelectionInput', 'MultipleChoiceInput'):
+        if self.id in (
+                'ItemSelectionInput', 'MultipleChoiceInput', 'DragAndDropSort'):
```

Comment:
You'll also need to add the suffix `Input` since that's the convention that we use to denote interactions here and elsewhere in the code.

Reply:
Ok, I'll change once these errors will be resolved. I need to change all the file names.

Reply:
This is still remaining.